kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -105,13 +129,9 @@ spec:
 
   resources:
     requests:
-      memory: 256Mi
+      memory: 128Mi
```

Comment:
for LA we might be better off to keep larger limits or remove them and we add those in GA to be save

Reply:
fixed